app/src/renderer/components/staking/ModalUnstake.vue

```diff
@@ -0,0 +1,108 @@
+<template lang="pug">
+  .modal-stake#modal-stake
+    //- Header
+    .stake-header
+      img.icon(class='stake-atom' src="~assets/images/cosmos-logo.png")
+      span.tm-modal-title Unstake
+      .tm-modal-icon.tm-modal-close(@click="close()")
+        i.material-icons close
+
+    //- To
+    tm-form-group.stake-form-group(
+      field-id='to' field-label='To')
+      tm-field#to(
+        readonly v-model="to")
+
+    //- Amount
+    tm-form-group.stake-form-group(
+      field-id='amount'
+      field-label='Amount'
+    )
+      tm-field#amount(
+        :max="maximum"
+        :min="0"
+        type="number"
+        v-focus
+        v-model="amount")
+
+    //- Footer
+    .stake-footer
+      tm-btn(
+        @click.native="close"
+        size="lg"
+        value="Cancel"
+      )
+      tm-btn(
+        @click.native="onUnstake"
+        color="primary"
+        size="lg"
+        value="Unstake"
+      )
+</template>
+
+<script>
+import Modal from "common/TmModal"
+import { TmBtn, TmField, TmFormGroup, TmFormMsg } from "@tendermint/ui"
+
+export default {
+  props: [`maximum`, `to`],
+  components: {
+    Modal,
+    TmBtn,
+    TmField,
+    TmFormGroup,
+    TmFormMsg
+  },
+  data: () => ({
+    amount: 0
+  }),
+  methods: {
+    close() {
+      this.$emit("update:showModalUnstake", false)
+    },
+    onUnstake() {
+      this.$emit(`submitUndelegation`, {
```

Comment:
Add invalid condition

Reply:
Won't do, as there is no outcome if the validation check fails